contributor_docs/access.md

```diff
@@ -0,0 +1,40 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means considering the vectors of diversity (e.g. gender, social, economic, race, ethnicity, language, ability, etc.) that can impact access/participation; and taking action to acknowledge, dismantle, and prevent barriers. We prioritize the needs of historically marginalized groups over the continued comfort of more privileged groups with p5.js.
```

Comment:
please change "ability" to "disability." this is preferred terminology when speaking about the disability community, as it foregrounds the "dis," which is the political part.

Reply:
thank you @agenusofmoth! made this change in ce00df9.